src/server/scripts/Commands/cs_lookup.cpp

```diff
@@ -959,18 +959,23 @@ class lookup_commandscript : public CommandScript
 
             bool known = target && target->HasSpell(id);
 
-            SpellEffectInfo const& spellEffectInfo = spellInfo->GetEffect(EFFECT_0);
-            bool learn = spellEffectInfo.IsEffect(SPELL_EFFECT_LEARN_SPELL);
-
-            SpellInfo const* learnSpellInfo = sSpellMgr->GetSpellInfo(spellEffectInfo.TriggerSpell, DIFFICULTY_NONE);
+            bool learn = false;
+            SpellInfo const* learnSpellInfo;
+            if (spellInfo->GetEffects().size()) {
```

Comment:
`{` always on next line according to our guidelines

Reply:
and i think !empty is more preferable